eZ/Bundle/PlatformInstallerBundle/src/Database/ConnectionFactory.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformInstallerBundle\Database;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+/**
+ * Decorator for \Doctrine\DBAL\Connection which injects custom Database Platform.
+ */
+class ConnectionFactory
```

Comment:
Before I created this class I tried to somehow reuse `\eZ\Bundle\EzPublishCoreBundle\ApiLoader\StorageConnectionFactory` or `\eZ\Publish\Core\Persistence\Doctrine\ConnectionHandler` but the first one obtained connection directly from Doctrine\DBAL services, the second one applies to layer we want to get rid of.

In fact this is a decorator for the `@ezpublish.persistence.connection` created by `\eZ\Bundle\EzPublishCoreBundle\ApiLoader\StorageConnectionFactory`.